### PR TITLE
OCPBUGS-36697, OCPBUGS-31832: csv align to 4.17 and remove webhook

### PR DESCRIPTION
**Pull request by karampok**

### CODE REVIEW

**Comment by karampok** [comment]
there is a problem with the published image from metallb
```
 omc -n metallb-system get pods
NAME                                                   READY   STATUS             RESTARTS   AGE
controller-77b9dfc9fd-72pbv                            1/2     ImagePullBackOff   0          7m
metallb-operator-controller-manager-759676786f-lzfnn   1/1     Running            0          8m
metallb-operator-webhook-server-664b8fb5bc-p58zl       1/1     Running            0          8m
speaker-2bp7d                                          4/6     ErrImagePull       0          7m
speaker-4lvp6                                          4/6     ImagePullBackOff   0          7m
speaker-6jxbf                                          4/6     ImagePullBackOff   0          7m
speaker-6rnqj                                          4/6     ImagePullBackOff   0          7m
speaker-flfsd                                          4/6     ImagePullBackOff   0          7m
speaker-k6p65                                          4/6     ImagePullBackOff   0          7m
```

**Comment by karampok** [comment]
the test is failing with image missing
```
omc -n openshift-metallb-operator get pods -o yaml|grep message
      message: 'containers with unready status: [manager]'
      message: 'containers with unready status: [manager]'
          message: Back-off pulling image "quay.io/openshift/origin-metallb-operator:4.17"
      message: 'containers with unready status: [webhook-server]'
      message: 'containers with unready status: [webhook-server]'
          message: Back-off pulling image "quay.io/openshift/origin-metallb:4.17"
```

File: manifests/stable/metallb.io_bgppeers.yaml [hunk 1]
```diff
@@ -6,18 +6,6 @@ metadata:
   creationTimestamp: null
   name: bgppeers.metallb.io
 spec:
-  conversion:
-    strategy: Webhook
-    webhook:
```
Comment (fedepaol) [inline manifests/stable/metallb.io_bgppeers.yaml:11]: Can you move this to a different commit, or even a different pr referencing the bug?

Comment (fedepaol) [reply manifests/stable/metallb.io_bgppeers.yaml:11]: * I see it's a separate commit. Let's just reference the bug in the pr then
